include/core/federated/net_util.h

```diff
@@ -38,9 +38,12 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef NET_UTIL_H
 #define NET_UTIL_H
 
+#ifndef PLATFORM_ARDUINO
```

Comment:
Idem

Reply:
Same as above.

Reply:
Here it would make sense to have it because Arduino would need to have its own implementations...